src/components/save.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import getMuiTheme from 'material-ui/styles/getMuiTheme';
+import Button from 'boundless-sdk/components/Button';
+import classNames from 'classnames';
+import enMessages from 'boundless-sdk/locale/en.js';
+import SaveIcon from 'material-ui/svg-icons/content/save';
+import {defineMessages, injectIntl, intlShape} from 'react-intl';
+import SaveDetailModal from './savedetailmodal';
+import {setMapConfig, saveMap} from '../reducers/actions'
+import { connectAdvanced } from 'react-redux'
+
+const messages = defineMessages({
+  savetext: {
+    id: 'savemap.savetext',
+    description: 'Text for Save Button',
+    defaultMessage: 'Save'
+  }
+});
+export class Save extends React.Component {
+  constructor(props, context) {
+    super(props);
+    this.state = {
+      muiTheme: context.muiTheme || getMuiTheme()
+    }
+  }
+  _openSaveModal() {
+    this.props.onClick();
```

Comment:
onClick not defined in the props? I now wonder what rules we enforce in eslint in this application, or we are not using it at all?